src/pages/demo.js

```diff
@@ -19,55 +19,10 @@ import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
 import github from 'prism-react-renderer/themes/github';
 import vsDark from 'prism-react-renderer/themes/vsDark';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
-import {
```

Comment:
We can remove the `react-instantsearch-dom` dependency if we don't use it anymore.

Reply:
It is removed